worli.sh

```diff
@@ -1,266 +1,259 @@
+    echo " "
+    echo "What's the path to 'WoR-PE_Package.zip'?"
+    read -r -p "[/*] E.g. '~/WoR-PE_Package.zip': " inst
+
     echo " "
-    
-    echo "what's the path to the 'Windows_ARM64_Drivers.zip'?"
-    read -r -p "[/*] eg '~/Windows_ARM64_Drivers.zip': " driv
-    
-    echo "- download the UEFI package: (not the source code)"
-    echo "  for Pi 4 and newer: https://github.com/pftf/RPi4/releases" 
-    echo "  for Pi 2, 3, CM3: https://github.com/pftf/RPi3/releases"
-    echo "  then rename the zip to 'UEFI_Firmware.zip'"
+    echo "- Download the driver package from: https://github.com/worproject/RPi-Windows-Drivers/releases (get the ZIP archive with the RPi prefix followed by your board version) and rename the ZIP file to Windows_ARM64_Drivers.zip"
     echo " "
-    
-    echo "what's the path to the 'UEFI_Firmware.zip'?"
-    read -r -p "[/*] eg '~/UEFI_Firmware.zip': " efi
-       ;;
+    echo "What's the path to 'Windows_ARM64_Drivers.zip'?"
+    read -r -p "[/*] E.g. '~/Windows_ARM64_Drivers.zip': " driv
+    ;;
+
     *)
- echo "Invalid input..."
- exit 1
- ;;
+
+    echo "Invalid input"
+    exit 1
+    ;;
 esac
 
 echo " "
-if [ -f $inst ]; then
-    echo "WoR-PE_Package.zip found"
-else 
-    echo "WoR-PE_Package.zip does not exist. abort."
+if [ -f $efi ]; then
+    echo "'UEFI_Firmware.zip' found"
+else
+    echo "'UEFI_Firmware.zip' does not exist. Abort."
     exit 1
 fi
-
 if [ -f $driv ]; then
-    echo "Windows_ARM64_Drivers.zip found"
-else 
-    echo "Windows_ARM64_Drivers.zip does not exist. abort."
+    echo "'Windows_ARM64_Drivers.zip' found"
+else
+    echo "'Windows_ARM64_Drivers.zip' does not exist. Abort."
     exit 1
 fi
-
-if [ -f $efi ]; then
-    echo "UEFI_Firmware.zip found"
-else 
-    echo "UEFI_Firmware.zip does not exist. abort."
+if [ -f $inst ]; then
+    echo "'WoR-PE_Package.zip' found"
+else
+    echo "'WoR-PE_Package.zip' does not exist. Abort."
     exit 1
 fi
 
 echo " "
-echo "prerequisites"
-echo " "
-echo "- get the windows iso: https://www.worproject.ml/guides/getting-windows-images"
-echo "  rename the iso to 'win.iso'"
+echo "Prerequisites"
 echo " "
-echo "- (NOT RECOMMENDED) if you want use use a custom wim rename it to 'install.wim' NOTE: you still need the iso the wim came from"
+echo "- Get the windows ISO: https://worproject.ml/guides/getting-windows-images"
+echo "- Rename the ISO file to 'win.iso'"
 echo " "
-echo "- If you're using a Raspberry Pi 4, you must update the bootloader to the latest version: https://www.raspberrypi.org/documentation/computers/raspberry-pi.html#updating-the-bootloader"
+echo "- (NOT RECOMMENDED) If you want use use a custom WIM, rename it to 'install.wim'. NOTE: You will still need the ISO where the WIM came from"
 echo " "
+echo "- If you're using a Raspberry Pi 4, you may have to update the Bootloader to the latest version: https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#updating-the-bootloader"
```

Comment:
hm mairo said must, perhaps its better to leave it like that?

Reply:
aight then (tho later versions of rpi4 comes with the latest bootloader already, that's why I'm saying that)